siriuspy/siriuspy/devices/device.py

```diff
@@ -198,6 +198,32 @@ def wait_float(
         func = _partial(isc, abs_tol=abs_tol, rel_tol=rel_tol)
         return self.wait(propty, value, comp=func, timeout=timeout)
 
+    def wait_several(self, props_values, timeout=None, comp='eq'):
+        """Wait several properties to reach the desired value.
+
+        Args:
+            props_values (list[2-tuple]): List of 2-tuples with property names
```

Comment:
Inconsistent type for `props_values` in docstring. Arg is used as dict in code.

Reply:
Indeed the description was wrong. This property was supposed to be a dictionary. Solved!